vello/src/wgpu_engine.rs

```diff
@@ -360,8 +386,11 @@ impl WgpuEngine {
                     transient_map
                         .bufs
                         .insert(buf_proxy.id, TransientBuf::Cpu(bytes));
-                    let usage =
-                        BufferUsages::COPY_SRC | BufferUsages::COPY_DST | BufferUsages::STORAGE;
+                    // TODO: restrict VERTEX usage to "debug_layers" feature?
```

Comment:
Yeah, we have lots of issues with implicit usages; I think we might need to either make this explicit at the recording level, or else loop through the recording and determine what capabilities we need. I think I'd prefer to just make it explicit.

But this isn't a task for this PR.